.github/workflows/phpcpd.yml

```diff
@@ -14,6 +14,15 @@ on:
       - '**.php'
       - '.github/workflows/phpcpd.yml'
   workflow_call:
+    inputs:
+      dirs:
+        description: Directories to check
+        required: true
+        type: string
+      options:
+        description: Options to phpcpd
+        required: true
```

Comment:
Why are options required?

Reply:
To add `exclude` files.
See https://github.com/codeigniter4/shield/pull/813